### PR TITLE
Updates Solaris platforms

### PR DESCRIPTION
**Pull request by kagarmoe**

### CODE REVIEW

File: chef_master/source/platforms.rst [hunk 1]
```diff
@@ -48,7 +48,7 @@ The following table lists the Foundational platforms for the chef-client and oha
      - ``5``, ``6``, ``7``
    * - Solaris
      - ``sparc``, ``x86``
-     - ``10``, ``11``
+     - ``10``, ``10 1\13`` (``10 U11``), ``11.0``, ``11.2``, ``11.3``
```
Comment (scotthain) [inline chef_master/source/platforms.rst]: Not sure how the formatting ends up with rst docs, but the end result for these should read something like:
```
10 1/13 ("U11"), 11.2, 11.3
```

**Review by scotthain — DISMISSED** [review]
Let me know if you have any questions regarding these!

File: chef_master/source/platforms.rst (hunk 1 above)
Comment (kagarmoe) [reply chef_master/source/platforms.rst]: Yep.  To clarify, are we supporting 11.0?

File: chef_master/source/platforms.rst [hunk 2]
```diff
@@ -48,7 +48,7 @@ The following table lists the Foundational platforms for the chef-client and oha
      - ``5``, ``6``, ``7``
    * - Solaris
      - ``sparc``, ``x86``
-     - ``10``, ``10 1\13`` (``10 U11``), ``11.0``, ``11.2``, ``11.3`` 
+     - ``10``, ``10 1\13`` (``"10U11"``), ``11.2``, ``11.3``
```
Comment (scotthain) [inline chef_master/source/platforms.rst]: I think there's an extra `10` in here

**Pull request merged**